provisioning/miniprov/hare_mp/main.py

```diff
@@ -225,6 +227,42 @@ def init(args):
         raise RuntimeError(f'Error while initializing cluster :key={error}')
 
 
+def start_mkfs(hostname: str, hare_config_dir: str):
+    motr_config_dir = '/etc/motr'
```

Comment:
This path needs to be updated according to the new conf-store key, kindly add a TODO here.

Reply:
done